### DISCUSSION TITLE
GSOC 2023

### DISCUSSION THREAD

**Discussion by parteekcoder** [GMOD/jbrowse-components#3472] [General]
Is GMOD participating in GSOC this year

**Comment by cmdcolin** [comment]
Someone in our chat room confirmed that the Open Genome Informatics (OBF) organization (which GMOD is a part of) should have time to apply for GSOC this year, and we can keep this thread updated if there is any news

**Reply by cmdcolin** [reply]
projects are listed here now :) http://gmod.org/wiki/GSOC_Project_Ideas_2023
